framework/bin/run_coverage.pl

```diff
@@ -30,7 +30,7 @@ =head1 NAME
 
 =head1 SYNOPSIS
 
-  run_coverage.pl -p project_id -d suite_dir -o out_dir [-f include_file_pattern] [-v version_id] [-t tmp_dir] [-D] [-A]
+  run_coverage.pl -p project_id -d suite_dir -o out_dir [-f include_file_pattern] [-v version_id] [-t tmp_dir] [-i instrument_classes] [-D] [-A]
```

Comment:
The flags -i and -A are contradictory when set at the same time. What about [-A | -i instrument_classes] -- the script should issue an error if both flags are set at the same time.